src/app/Dashboard/Dashboard.tsx

```diff
@@ -36,20 +36,82 @@
  * SOFTWARE.
  */
 import * as React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
 import { TargetView } from '@app/TargetView/TargetView';
+import { getFromLocalStorage, saveToLocalStorage } from '@app/utils/LocalStorage';
+import { AddCard } from './AddCard';
+import { DashboardCardActionMenu } from './DashboardCardActionMenu';
 import { AutomatedAnalysisCard } from './AutomatedAnalysis/AutomatedAnalysisCard';
-import { Grid, GridItem } from '@patternfly/react-core';
+
+export interface CardConfig {
+  component: React.FunctionComponent;
+  props?: React.PropsWithChildren<any>;
+}
 
 export interface DashboardProps {}
 
+export interface DashboardCardProps {
+  actions?: JSX.Element[];
+}
+
+export const DashboardCards: CardConfig[] = [
+  {
+    component: AutomatedAnalysisCard,
+    props: {
+      isLarge: true,
```

Comment:
Look like if we set this to true. There are some misalignments between toolbar and content in term of padding in the automatic analysis card.

![Screenshot from 2022-12-08 18-54-34](https://user-images.githubusercontent.com/68053619/206592315-b441b5ad-9bd6-41e7-9a99-51e19b43a4ad.png)


We guess we can keep `isCompact` or add more padding to the toolbar (i.e. maybe around 1rem)?